Utils/Multi Language Graphic Utils.cpp

```diff
@@ -203,6 +212,15 @@ BOOLEAN GetMLGFilename( SGPFILENAME filename, UINT16 usMLGGraphicID )
 			case MLG_PREBATTLEPANEL:
 				sprintf( filename, "GERMAN\\PreBattlePanel_german.sti" );
 				return TRUE;
+			case MLG_PREBATTLEPANEL_800x600:
+				sprintf(filename, "INTERFACE\\PreBattlePanel_800x600_german.sti");

```

Comment:
The correct path should be GERMAN\\PreBattlePanel_800x600_german.sti, similar to the original prebattle sti file. Right now the game will error out on the assert at line 546 in PreBattle Interface.cpp as it fails to fails to find the file

Reply:
Thanks, resolved